src/CommonDBTM.php

```diff
@@ -467,14 +467,22 @@ public function post_getFromDB()
      */
     public function showForm($ID, array $options = [])
     {
+        /** @var array $CFG_GLPI */
+        global $CFG_GLPI;
+
         $this->initForm($ID, $options);
         $new_item = static::isNewID($ID);
         $in_modal = (bool) ($_GET['_in_modal'] ?? false);
+        if (!$new_item && (in_array($this->getType(), $CFG_GLPI['cluster_types']))) {
+            $additional_var = [
+                'cluster' => Cluster::getClusterByItem($this)
+            ];
+        }
```

Comment:
`cluster` should always be apssed to the template. It is easier to read/understand/maintain.

```suggestion
        $cluster = !$new_item && in_array(static::class, $CFG_GLPI['cluster_types'], true)
            ? Cluster::getClusterByItem($this)
            : null
```